docs/apis/topaz_cli.md

```diff
@@ -696,88 +695,207 @@ Parameter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Abbr.  | Description
+                                           If not defined, it will be set
+                                           to the same value as the file
+                                           parameter
+ -R,--recursive                            Recursively find all relevant
+                                           test scenarios
+ -S,--source-folder <arg>                  Optional file path to a folder
+                                           that contains source code of
+                                           tested programs. Default value
+                                           is 'cobol'. Is only used to set
+                                           the source path in code
+                                           coverage reports.
+ -s,--server <arg>                         Address to the CES Server, e.g.
+                                           https://server.topaztotaltest.c
+                                           om/totaltestapi/. Can be left
+                                           out which will result in no
+                                           repository usage
+ -sid,--siteid <arg>                       Site Id used for cloud license
+                                           check
+ -u,--userid <arg>                         tso userid
+ -U,--use-scenario-files                   Find scenario files instead of
+                                           context files and auto generate
+                                           a context file for the selected
+                                           environment for each scenario
+                                           found
+ -v,--sonar-version <arg>                  Sonar Qube version - either 5
+                                           or 6. Default value is 6
+ -x,--upload-result                        Upload result to server
 ```
-The CLI interface will generate a log file in the logs directory in the location specified by the workspace directory. This log file can be used to track down and understand issues in an execution.
-<!--stackedit_data:
-eyJoaXN0b3J5IjpbLTIxMjUzMTQ3OF19
--->
\ No newline at end of file
+The --enviroment and the --file arguments are the primary ones, and the --userid and --password arguments for the TSO credential to be used. If the --file argument is relative, the --root-folder parameter is required. It is used to calculate the relative path to the folder being executed and this path is set at the result in the Functional Test server as the External Reference (if --upload-result option is used). Normally the root folder will be the folder to which files have been checked out from version control. With the Jenkins plugin the root folder will be set to the Jenkins workspace unless the --file path is absolute.
+Sonar cube and JUnit output files are also generated and placed in the TTTSonar and TTTUnit directories relative to the root folder given in the --root-folder parameter.
```

Comment:
It should be "SonarQube" instead of "Sonar cube"

Reply:
fixed